src/components/Dashboard/TodoList.vue

```diff
@@ -600,27 +618,56 @@ export default {
       }
     },
 
+    /** Loads a NEW Annual Report todo. */
+    loadAnnualReportTodo (task) {
+      const todo: FilingIF = task.task.todo
+      if (todo?.header && todo?.business) {
+        const ARFilingYear: number = todo.header.ARFilingYear
+        const subtitle: string = task.enabled && !this.isBComp ? '(including Address and/or Director Change)' : null
+        // nextAnnualReport (and nextArDate and arDueDate below) are only used for BCOMP ARs
+        const nextArSimpleDateTime: string = this.apiToSimpleDateTime(todo.business.nextAnnualReport)
```

Comment:
This is where the BCOMP AR gets the Next Annual Report date from. (See also Next AR Date and AR Due Date below.)

And by the way, this is the same year as AR Filing Year. (I had to fix some unit tests.)